pending/17-handlebars.pod

```diff
@@ -0,0 +1,84 @@
+=head1 Dancer2::Template::Handlebars
+
+I know that our editor-in-chief this year, the amazing L<Cromedome|https://metacpan.org/author/CROMEDOME>, is hoping
+for some deep wisdom here about why and how I created Dancer2::Template::Handlebars.
```

Comment:
It would seem sensible to make this a link for people to easily click through to find the plugin on CPAN I think?

Reply:
Easy to do, and done.